app/scripts/utils/data-utils.ts

```diff
@@ -0,0 +1,24 @@
+import { VedaData, VedaDatum, DatasetData } from '$types/veda';
+
+export const findParentDataset = ({
+  datasets,
+  datasetId
+}: {
+  datasets: DatasetData[];
+  datasetId: string;
+}): DatasetData | undefined => {
+  const parentDatset = datasets.find(
```

Comment:
Could we incorporate this one line `datasets.find(...)` directly into the `useParentDataset` hook? I don't see a reason why we need to keep it as a separate utils function. Is this even used elsewhere? Seems you deleted the places where it was used in data-catalog.

Reply:
Yes! great idea.